bin/setup_venv.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Run this from the project directory
+# Downloads python 2.7.9
+# Install it in the project directory (where we have write access)
+# Load a venv with this python
+# Loads other dependencies into the venv.
+
+# Get present working directory
+pwd=$(pwd)
+
+# Get Python sources and compile
+wget https://www.python.org/ftp/python/2.7.9/Python-2.7.9.tgz
+tar xfz Python-2.7.9.tgz
+cd Python-2.7.9/ || exit
+make clean
+./configure --prefix="$pwd"/.localpython
+make
+make install
+cd ..
+
+# Test if the version works
```

Comment:
If this test is for the user to see what version it is, it should stand out more in the output. It could also be automated, and exit if it's the wrong version.